solr/solrj/src/test/org/apache/solr/client/solrj/impl/BasicHttpSolrClientTest.java

```diff
@@ -478,6 +484,8 @@ public void testUpdate() throws Exception {
       assertEquals("\u1234", DebugServlet.parameters.get("a")[0]);
 
       // XML response and writer
+      // Have not been able to move this to the Builder pattern, the below check for application/xml
+      // always returns as application/javabin when I try this!
```

Comment:
Would be good to dig to why this happens

Reply:
yeah, I'd love a fresh set of eyes, I don't get why it doesn't work..   maybe somethign about the `DebugServlet` or some sort of something being reused...